### PR TITLE
BUGFIX: Renaming adjustments

### PR DESCRIPTION
**Pull request by aertmann**

### CODE REVIEW

File: Migrations/Mysql/Version20121030221352.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ public function up(Schema $schema)
         $this->abortIf($this->connection->getDatabasePlatform()->getName() != "mysql");
 
         $this->addSql("ALTER TABLE typo3_phoenixdemotypo3org_domain_model_registration RENAME TO typo3_neosdemotypo3org_domain_model_registration");
-        $this->addSql("UPDATE typo3_neos_domain_model_site SET nodename = 'neosdemotypo3org', siteresourcespackagekey = 'Neos.Demo' WHERE nodename = 'phoenixdemotypo3org'");
+        $this->addSql("UPDATE typo3_neos_domain_model_site SET nodename = 'neosdemotypo3org', siteresourcespackagekey = 'TYPO3.NeosDemoTypo3Org' WHERE nodename = 'phoenixdemotypo3org'");
```
Comment (radmiraal) [inline Migrations/Mysql/Version20121030221352.php]: this is a rollback of eager renaming before?

Comment (aertmann) [inline Migrations/Mysql/Version20121030221352.php]: yep

**Comment by radmiraal** [comment]
didn't give it a spin, but looks good by reading

**Comment by aertmann** [comment]
should suffice, thanks

**Comment by kdambekalns** [comment]
Well, shouldn't this have gone into 2.2, now the 2.2 `Sites.xml` still has `neosdemotypo3org` as site node name… @aertmann

**Comment by aertmann** [comment]
@kdambekalns right it should, didn't know there was a 2.2 branch at that point since there wasn't one for neos-development-collection meaning it had been created manually and not using the branch script.. anyway I'll backport the changes

**Pull request merged**